.github/workflows/main.yml

```diff
@@ -99,6 +99,14 @@ jobs:
           curl -sL "https://sourceforge.net/projects/nsis/files/NSIS%203/${nsis_version}/nsis-${nsis_version}-log.zip/download" -o "nsis-${nsis_version}-log.zip"
           7z x "nsis-${nsis_version}-log.zip" -aoa -o"$CONDA_PREFIX/NSIS/"
           echo "NSIS_USING_LOG_BUILD=1" >> $GITHUB_ENV
+      - name: Set temp dirs correctly
+        if: startsWith(matrix.os, 'windows')
+        # https://github.com/actions/virtual-environments/issues/712
+        shell: powershell
+        run: |
+          echo "TMPDIR=$env:USERPROFILE\AppData\Local\Temp" >> $env:GITHUB_ENV
+          echo "TEMP=$env:USERPROFILE\AppData\Local\Temp" >> $env:GITHUB_ENV
+          echo "TMP=$env:USERPROFILE\AppData\Local\Temp" >> $env:GITHUB_ENV
```

Comment:
Are you sure this is necessary? We recently removed all of this from conda-build

Reply:
I did read the referenced github issue and there still seem to be reasons to set those tmp variables... if we actually need that here is unclear to me. we can commit your suggestion to remove it and revert in case thats failing CI then.